tool/gravity/cli/config.go

```diff
@@ -436,6 +450,27 @@ func (i *InstallConfig) RunLocalChecks() error {
 	}))
 }
 
+// BootstrapSELinux installs the default SELinux policy on the node
+func (i *InstallConfig) BootstrapSELinux(printer utils.Printer) error {
+	if !i.SELinux || i.FromService {
+		return nil
+	}
+	metadata, err := monitoring.GetOSRelease()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	if !selinux.IsSystemSupported(metadata.ID) {
+		log.WithField("os", metadata).Info("No SELinux policy for specified system, will skip.")
```

Comment:
Will an error be returned if you attempt to launch it with selinux on an unsupported distro? Or it will silently fallback to "no selinux"?

Reply:
Since the knob is on automatically, I thought it is wrong to fail with an error - it flips it and continues as if it has been disabled.